omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -202,14 +253,72 @@ code from the project's GitHub account to build locally::
     :doc:`/developers/build-system`
         Developer documentation page on how to build the OMERO.server
 
+3. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/projects/openmicroscopy/dist`.
+   Prepend the :file:`bin` directory to your :envvar:`PATH`::
+
+    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
 
+   and follow the steps for setting up the database and OMERO data directory as mentioned in the previous section.
 
-Then prepend the development :file:`bin` directory to your :envvar:`PATH` to
-pick the right executbale::
+4. Activate the Virtualenv enviornment that we created earlier in the "Requirements"
+   section::
 
-    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
+    $ source ~/Virtual/Omero/bin/activate
+
+5. Install Python dependencies using pip::
+
+    $ pip install -r ~Omero/server/share/web/requirements-py27-all.txt
+    $ cd /usr/local
+    $ bash bin/omero_python_deps
+
+
+OMERO configuration
+-------------------
+
+1. Start the database server::
+
+    $ pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w start
```

Comment:
Does not work for me. From the log:
``could not open directory "pg_tblspc": No such file or directory``